### ISSUE TITLE
Autostart does not execute user scripts at startup

### ISSUE THREAD

**Issue by Maxquig** [feren-OS/Bug-Reporting-Center#287]
**Package Name or Section of Feren OS where this bug resides**
Where does this bug happen? (e.g.: feren-themer-cinnamon) ___Autostart Login Scripts

**Describe the bug**
When using HDMI-0 monitor with a HDMI cable, the default AMD Kaveri Graphic driver for my AMD A10-7700K Radeon CPU/GPU does not support corrections to overscanning and underscanning of my 1920x1080 resolution.
I can execute the command: xrandr --output HDMI-0 --panning 1920x1080 --transform 1.085,0,-62,0,1.07,-21,0,0,1.02 to temporarily adjust my monitor's settings. But I want to run that command automatically at startup. So I wrote that command into a bash shell script.

**Steps to Reproduce**
1. Start computer using HDMI cable for monitor.
2. Select multi-boot to launch FEREN OS
3. FEREN OS opens with part of the image off from the edges all around the screen.
4. Manually open a terminal window and run the script "fixoverscan". It looks like this
5. #!/bin/bash
xrandr --output HDMI-0 --panning 1920x1080 --transform 1.085,0,-62,0,1.07,-21,0,0,1.02
6. Just like magic, the image on the monitor is fixed.
7. Launch System Settings - Autostart
8. Click ADD button
9. Try both options, Add Application and Add Login Script
10. Browse to the bash script file "fixoverscan"
11. It appears to be all set up.
12. Restart the system on FEREN OS
13. FAIL!
14. The image runs off the edge all the way around.
15. Run the script from the executable file "fixoverscan"
16. Image gets adjusted perfectly.

**Expected behavior**

**Screenshots**

**Additional context**
A quick WEB search indicates Fedora is the source of the bug, but I could find no evidence to indicate Fedora has any plans to fix it.

**Comment by dominichayesferen** [comment]
Is the file marked as executable?

**Comment by Maxquig** [comment]
Yes, It is executable by anyone. In fact I execute it in a terminal window to fix my monitor settings. It works well. It seems that EVERY disto but Linux Mint has the same problem. I've tried 12 distros recently.
    On Saturday, November 5, 2022 at 12:14:26 PM CDT, Dominic Hayes - Feren OS Dev ***@***.***> wrote:  
 
 


Is the file marked as executable?

**Comment by Maxquig** [comment]
I think I solved the problem I had. It turns out Autostart did start the script. It was a matter of timing. I added another script that merely launched kcalc. It launched just fine. So I guessed that it was launching my script before Xserver was up and running. So I added a 5 second delay (sleep 5) to the top of my script. That fixed it. I'm running Kubuntu right now, but later today I'll do the same in Feren OS. I'm confident it will work. Then I'll close this bug.
